tests/lib/openstack_query/query_blocks/test_result_container.py

```diff
@@ -0,0 +1,300 @@
+from unittest.mock import MagicMock, patch, call, NonCallableMock
+import pytest
+
+from exceptions.query_chaining_error import QueryChainingError
+from openstack_query.query_blocks.results_container import ResultsContainer
+from tests.lib.openstack_query.mocks.mocked_props import MockProperties
+
+
+@pytest.fixture(name="setup_instance_with_results")
+def setup_instance_with_results_fixture():
+    """
+    Returns an instance of ResultContainer with mocked results
+    """
+
+    def _setup_instance_with_results(mock_results):
+        val = ResultsContainer(prop_enum_cls=MockProperties)
+        # pylint:disable=protected-access
+        val._results = mock_results
+        return val
+
+    return _setup_instance_with_results
+
+
+def test_to_props_results_empty(setup_instance_with_results):
+    """
+    Test to_props method when results are empty - return empty list
+    """
+    instance = setup_instance_with_results([])
+    instance.to_props(MockProperties.PROP_1, MockProperties.PROP_2)
+
```

Comment:
What is the pass/fail condition in this test? I cannot see if there is an assert in this or either called methods

Reply:
good catch - I forgot to add an assertion